R/drive_download.R

```diff
@@ -2,8 +2,10 @@
 #'
 #' @template file
 #' @param out_path Character. Path for output file. Will default to its Google Drive
-#'   name. Google files must include the intended extension and can be downloaded to types
+#'   name. Google files must include the intended extension. This can be specified by the
+#'   `out_path` or the `type` parameter. Google files can be downloaded to types
 #'   specified in the [Drive API documentation](https://developers.google.com/drive/v3/web/manage-downloads).
+#' @param type Character. Extension you would like for the local file.
```

Comment:
Could this be another case of "anything that `drive_mime_type()` can handle? Yeah, I see it will almost always be a file extension, but I don't see any downside to running this through `drive_mime_type()`. If you agree, I have wording to that effect in some other help file, which you could search for.

The wording re `out_path` still seems a bit confusing. You don't have to include the file extension or even `out_path` at all, if you specify `type`, right? Also I suspect "Google files" is not yet clear enough that it means Sheets, Docs, and the like.